beaker-tests/DockerTestEnv/Makefile

```diff
@@ -13,7 +13,7 @@ bld: build
 run:
 	$(CONTAINER_ENGINE) run \
 		-dit \
-		-v $(mkfile_path)/../..:/root/copr \
+		-v $(mkfile_path)/../..:/root/copr:Z \
```

Comment:
Would `:z` work too? I generally try to avoid `:Z` because the bindmount cannot be accessed from other containers. Shouldn't be a problem here but still

Reply:
That was the reason why I put `:Z` here to pretend at least I care about minimal privileges since I know we don't want to access the bind-mount from other containers. But yeah, it doesn;t mater in this use case so `:z` would work too.